userdocs/src/index.md

```diff
@@ -91,7 +91,7 @@ able to use `kubectl`. You will need to make sure to use the same AWS API creden
 dependencies installed already.
 
 To learn more about how to create clusters and other features continue reading the
-[usage section](usage/creating-and-managing-clusters/).
+[Creating and Managing Clusters section](usage/creating-and-managing-clusters).
```

Comment:
I couldn't find a better way to link this file and use the direct path :/ so I added this excluded for now since the link checker tries to find the file in markdown and fails >.<'